lib/inferno/terminology/expected_manifest.yml

```diff
@@ -678,15 +696,23 @@
   - http://hl7.org/fhir/narrative-status
 - :url: http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1099.30
   :file: cts_nlm_nih_gov_fhir_ValueSet_2_16_840_1_113762_1_4_1099_30.msgpack
-  :count: 124
+  :count: 590
   :type: bloom
-  :code_systems: []
+  :code_systems:
+  - http://snomed.info/sct
+  - http://terminology.hl7.org/CodeSystem/v3-ParticipationFunction
 - :url: http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1010.6
   :file: cts_nlm_nih_gov_fhir_ValueSet_2_16_840_1_113762_1_4_1010_6.msgpack
-  :count: 172
+  :count: 196
   :type: bloom
   :code_systems:
   - http://hl7.org/fhir/sid/cvx
+- :url: http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1010.4
+  :file: cts_nlm_nih_gov_fhir_ValueSet_2_16_840_1_113762_1_4_1010_4.msgpack
+  :count: 1000
```

Comment:
This is an oddly round number, not a cap?

Reply:
It's not a cap. We have plenty of larger value sets.